src/api_server/swagger/firecracker.yaml

```diff
@@ -1227,4 +1227,4 @@ definitions:
         description: Path to UNIX domain socket, used to proxy vsock connections.
       vsock_id:
         type: string
-        description: This parameter has been deprecated since v1.1.0.
+        description: This parameter has been deprecated since v1.1.1.
```

Comment:
`vsock_id` was deprecated in v1.0.0: https://github.com/firecracker-microvm/firecracker/blob/main/CHANGELOG.md#changed-1.